fail2ban/tests/servertestcase.py

```diff
@@ -158,6 +158,9 @@ class Transmitter(TransmitterBase):
 	def setUp(self):
 		self.server = TestServer()
 		super(Transmitter, self).setUp()
+		# so far isStarted only tested but not used otherwise
+		# and here we don't really .start server
+		self.assertFalse(self.server.isStarted())
```

Comment:
Why are you against more testing? ;-)

On July 13, 2016 3:26:33 AM EDT, "Serg G. Brester" <notifications@github.com> wrote:
>> @@ -158,6 +158,9 @@ class Transmitter(TransmitterBase):
>>  	def setUp(self):
>>  		self.server = TestServer()
>>  		super(Transmitter, self).setUp()
>> +		# so far isStarted only tested but not used otherwise
>> +		# and here we don't really .start server
>> +		self.assertFalse(self.server.isStarted())
>
>I think it will be enough to check isStarted once (see below in
>testStartFailedSockExists)...
>Additionally I'm against asserts in setup of the test cases (executed
>in each test)... so I will outsource it in separate case...
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/sebres/fail2ban/pull/7/files/dfcf6f1963b87a7fbe87a78f47ba5c0ac40dd558#r70575815